packages/shared/src/hooks/referral/useReferralConfig.ts

```diff
@@ -17,19 +17,29 @@ export const useReferralConfig = ({
     { url: `https://og.daily.dev/api/refs/${referringUser.id}` },
   ];
 
-  if (campaign === ReferralCampaignKey.Search) {
-    return {
-      title: defaultTitle,
-      description: defaultDescription,
-      images: defaultImages,
-      redirectTo: '/my-feed',
-    };
-  }
+  switch (campaign as ReferralCampaignKey) {
+    case ReferralCampaignKey.Search:
+      return {
+        title: defaultTitle,
```

Comment:
Maybe we can abstract the default to a const, and use {...default, overwrite: something } instead?
It will avoid having the same lines 3 times.